src/Admin/BaseBlockAdmin.php

```diff
@@ -273,6 +273,10 @@ protected function configureDatagridFilters(DatagridMapper $datagridMapper)
      */
     private function loadBlockDefaults(BlockInterface $block)
     {
+        if (null === $this->blockManager->has($block->getType() ?? '')) {
+            return $block;
+        }
```

Comment:
null === $block->getType() || !$this->blockManager->has($block->getType())

I think you missed your changes